lnrpc/rpc.proto

```diff
@@ -1282,15 +1310,35 @@ message PendingChannelsResponse {
 
 message WalletBalanceRequest {
 }
-message WalletBalanceResponse {
+message ChainWalletBalance{
+
+    string chain = 1 [json_name = "chain"];
     /// The balance of the wallet
-    int64 total_balance = 1 [json_name = "total_balance"];
```

Comment:
Comment should be changed to something like "the balance on the chain" . Similar changes for the comments on other fields below.